web/src/queries/storage/config-model.ts

```diff
@@ -104,6 +116,20 @@ function disableBoot(originalModel: configModel.Config): configModel.Config {
   return setBoot(originalModel, { configure: false });
 }
 
+function deletePartition(
+  originalModel: configModel.Config,
+  driveName: string,
+  mountPath: string,
+): configModel.Config {
+  const model = copyModel(originalModel);
+  const drive = findDrive(model, driveName);
+  if (drive === undefined) return;
+
+  const partitions = (drive.partitions || []).filter((p) => p.mountPath !== mountPath);
+  drive.partitions = partitions;
```

Comment:
NP: I know the PR still in draft mode, but just in case you overlooked it this looks like a good place for using the `findPartition` method you added above.

Reply:
Huh, I don't think I can use that method... the lambdas differ in `===` and `!==`... or did you mean something else?

Reply:
Actually I didn't realize about such a difference :) Go ahead.